src/web/mjs/connectors/Boosei.mjs

```diff
@@ -0,0 +1,37 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class Boosei extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'boosei';
+        super.label = 'Boosei';
+        this.tags = [ 'webtoon', 'indonesian' ];
+        this.url = 'https://boosei.com';
+        this.path = '/manga/list-mode/';
+    }
+
+    async _getPages(chapter) {
+        const script = `
+            new Promise((resolve, reject) => {
+                if(window.ts_reader) {
```

Comment:
Shouldn't this be `window. ts_reader_control`?

Reply:
ts_reader_control being called by ts_reader.run 
![image](https://user-images.githubusercontent.com/31276236/102004379-b80e1a80-3d42-11eb-85fd-3ccc95a6b67e.png)

Reply:
The reason for the condition is to ensure that `ts_reader_control` exist, but the condition checks if `ts_reader` exists. It doesn't matter how `ts_reader_control` is created or initialized, it matters that it is not undefined before accessing it.

The global variable `ts_reader_control` is declared somewhere outside of `ts_reader.run`, so the method call is not related to the variable declaration.

Reply:
Maybe the condition check is not even required, can this be removed along with the whole `else` block?

In case `ts_reader_control` is expected to exists on all chapters, then there is no reason to provide the alternative HTML extraction method.